BookingsAPI/Bookings.API/Bookings.API.csproj

```diff
@@ -20,6 +20,7 @@
     <UseAppHost>false</UseAppHost>
   </PropertyGroup>
   
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'BookingsApi' " />
```

Comment:
What does this do?

Reply:
Think this is something that VS did by itself

Reply:
Could you remove it please, it's not needed